template/README.md

```diff
@@ -770,15 +770,34 @@ Note that tests run much slower with coverage so it is recommended to run it sep
 
 By default `npm test` runs the watcher with interactive CLI. However, you can force it to run tests once and finish the process by setting an environment variable called `CI`. Popular CI servers already set it by default but you can do this yourself too:
 
-#### Windows (cmd.exe)
+### On CI servers
+#### Travis CI
+
+1. Following the [Travis Getting started](https://docs.travis-ci.com/user/getting-started/) guide for syncing your Github repository with Travis.  You may need to initialize some settings manually in your [profile](https://travis-ci.org/profile) page.
+1. Add a `.travis.yml` file to your git repository.
+```
+language: node_js
+node_js:
+  - 6
+cache:
+  directories:
+  - node_modules
+script
+  - CI=true npm test
```

Comment:
Thanks @ayrton , I removed CI=true in updated commit.  Also I added back node 4 tests as the issue is fixed in the new version